patches/chrome-browser-ui-BUILD.gn.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/chrome/browser/ui/BUILD.gn b/chrome/browser/ui/BUILD.gn
+index 72dde06a99a77b34f466cbfba3d478dc60b2d86c..ec163151304d80d26b8d854bb288d17d4e1164de 100644
+--- a/chrome/browser/ui/BUILD.gn
++++ b/chrome/browser/ui/BUILD.gn
+@@ -1097,6 +1097,11 @@ split_static_library("ui") {
+     "//ui/webui",
+     "//v8:v8_version",
+   ]
++  if (brave_chromium_build) {
++    deps += [
++      "//brave/browser/ui",
++    ]
++  }
+   allow_circular_includes_from +=
+       [ "//chrome/browser/ui/webui/bluetooth_internals" ]
+ 
+@@ -2661,6 +2666,7 @@ split_static_library("ui") {
+       "//ui/events:dom_keycode_converter",
+     ]
+ 
++    if (!brave_chromium_build) {
```

Comment:
isn't this redundant? brave_chrome_build will never be is_chrome_branded

Reply:
I used this because of `else` clause below.
This will disable `else` clause and all related things are handled in `brave/browser/ui/BUILD.gn` instead.

Reply:
but the else is just version_updater_basic, right? Do we really care if that gets built and not used? If it cases other problems then I think we should override the file to be empty in chromium_src

Reply:
I think basic implementation would be needed in non-official build, too.
In `AboutHandler`, it always reference it

Reply:
then can't we add it just like we did with google_update_win.cc ?

Reply:
`google_update_win` was only included in `google_chrome_build`.
So, I don't need to touch that.
But, `VersionUpdaterBasic` was included in non `google_chrome_build`.

Reply:
yep, I understand now, please just modify the patch as discussed

Reply:
done.